index.js

```diff
@@ -48,7 +50,7 @@
           // thinking in terms of Dom node nesting, running counter
           // to React's "you shouldn't care about the DOM" philosophy.
           while(source.parentNode) {
-            found = (source === localNode);
+            found = (source === localNode || source.className.match(IGNORE_CLASS_REGEXP));
```

Comment:
classList lets us as targeted questions, so we can do this:
```
source.classList.contains(IGNORE_CLASS)
```
without needing to match regexp on the HTML node "class" node attribute values.